.github/ISSUE_TEMPLATE/game_issue.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Bug report about game issues
+about: This report type is speficically meant or issues with games
+
+---
+
+**Game information**
+<!-- Name of the game or games that do not work -->
+
+**Distribution name**
+<!-- If your distribution is versioned, incldue the version. Also preferably last update time -->
+
+**Flatpak info**
+<!-- Flatpak version -->
```

Comment:
Perhaps include a command to run that includes the information you're interested in. Is just `flatpak --version` enough? Perhaps `flatpak --show-details --all` too?

Reply:
Doing `--all` might show a bit too much information since it would contain all the other flatpaks that are installed, is there a way to get just the Steam information and its deps?

Reply:
I guess could give a simple grep spell that would only give this app's namespace and everything under org.freedesktop.Platform

Reply:
`flatpak info com.valvesoftware.Steam` shows a lot of information about the Steam pak but I can't see anything to list the deps of it too.

Reply:
`glxinfo | grep OpenGL`? Or does something need to be run from the inside?

Reply:
Yes, definitely. We want to know which GL drivers are actually in use from runtime extensions

Reply:
To be honest, we could probably consider bundling glxinfo and vulkaninfo for helping get better information from inside the app

Reply:
If this is the case it would probably be worth adding a shell script too and a line here saying something like

Run `flatpak run --command=get-driver-inf com.valvesoftware.Steam` and paste the results here

Reply:
My main issue with this template is that it'll be very easy for people to read it and not know exactly what info is needed and will end up either not putting enough/correct information or way too much. If there's anything that can be added to the template and the flatpak then it'd help a lot.

Reply:
Yeah, this was not intended as a final thing, more as asking for suggestions. My Linux currently currently can't start gnome-shell at all so I've difficulty experimenting which commands to use.

Reply:
`flatpak --gl-driver` is probably good enough for driver version.